tests/models/wav2vec2/test_feature_extraction_wav2vec2.py

```diff
@@ -123,6 +123,14 @@ def test_call(self):
         for enc_seq_1, enc_seq_2 in zip(encoded_sequences_1, encoded_sequences_2):
             self.assertTrue(np.allclose(enc_seq_1, enc_seq_2, atol=1e-3))
 
+        # Test 2-D numpy arrays are batched.
+        speech_inputs = [floats_list((1, x))[0] for x in (800, 800, 800)]
+        np_speech_inputs = [np.asarray(speech_input) for speech_input in speech_inputs]
```

Comment:
This should all be a NumPy array to trigger our new condition `is_batched_numpy`! Otherwise we'll just trigger `is_batched` as before:
```suggestion
        np_speech_inputs = np.asarray(speech_inputs)
```
Note that we can call `np.asarray` on our entire `speech_inputs` list in one go since all the dims are the same